jgrapht-core/src/main/java/org/jgrapht/alg/matching/EdmondsBlossomShrinking.java

```diff
@@ -32,11 +32,14 @@
  *
  * @author Alejandro R. Lopez del Huerto
  * @since Jan 24, 2012
+ * @deprecated In favor of the more efficient {@link EdmondsMaximumCardinalityMatching} implementation.
  */
+@Deprecated
 public class EdmondsBlossomShrinking<V, E>
     implements MatchingAlgorithm<V, E>
 {
     private final Graph<V, E> graph;
+    private final MatchingAlgorithm<V,E> initializer;
```

Comment:
If deprecated, best to not make any changes in this file. Let us keep the old version, as deprecated.